mapiproxy/servers/default/nspi/dcesrv_exchange_nsp.c

```diff
@@ -345,6 +395,8 @@ static void dcesrv_NspiUpdateStat(struct dcesrv_call_state *dce_call, TALLOC_CTX
 {
 	struct emsabp_context		*emsabp_ctx = NULL;
 	bool				container_exists;
+        struct PropertyTagArray_r	*mids;
```

Comment:
Wrong indentation somewhere. Please stick to <TAB> or spaces. Whatever is chosen in the file.